apps/myaccount/src/components/consents/consent-edit.tsx

```diff
@@ -17,10 +17,17 @@
  */
 
 import { TestableComponentInterface } from "@wso2is/core/models";
+import _ from "lodash";
```

Comment:
Since you are only using `_chain` function, it's a better approach to import only that util function. By importing `_` you import the entire library. 

Reference on single lodash import - [link](https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark)

Reply:
I'll work on it and fix it ASAP. Thanks 💯

Reply:
Applied this improvement in commit a832ff4